app/smc/vuart.overlay

```diff
@@ -10,10 +10,15 @@
 		zephyr,shell-uart = &vuart0;
 	};
 
-	vuart0: uart_tt_virt {
-		compatible = "tenstorrent,vuart";
-		version = <0x00000000>;
-		status = "disabled";
+	vuarts {
```

Comment:
I guess this is necessary for non-contiguous / starting from 0 instances, right?